guide/yaml/advanced-example.md

```diff
@@ -177,4 +211,41 @@ The ELK blueprint also contains a good example of a child entity.
   - type: logstash-child
 ~~~
 
-In this example, a logstash-child is started as a child of the parent Tomcat server. The Tomcat server needs to be configured with a `children.startable.mode` to inform Brooklyn when to bring up the child. In this case we have selected background so that the child is disassociated from the parent entity, and late to specify that the parent entity should start before we start the child.
+In this example, a logstash-child is started as a child of the parent Tomcat server. The Tomcat server needs 
+to be configured with a `children.startable.mode` to inform Brooklyn when to bring up the child. In this case
+ we have selected background so that the child is disassociated from the parent entity, and late to specify that
+  the parent entity should start before we start the child.
+
+The example also shows how to configure Logstash inputs and filters, if necessary, for a particular application, 
+in this case Tomcat.
+~~~yaml
```

Comment:
it looks like you need a blank line in front of the `~~~` to have the block presented correctly